WebARonARKit/WebARonARKit/ViewController.m

```diff
@@ -608,24 +608,7 @@ - (void)session:(ARSession *)session didUpdateAnchors:(nonnull NSArray<ARAnchor
 
 - (void)session:(ARSession *)session didRemoveAnchors:(nonnull NSArray<ARAnchor *> *)anchors
 {
-  NSString *removedStr = @"[";
-  for (int i = 0; i < anchors.count; i++) {
-    if (![anchors[i] isKindOfClass:[ARPlaneAnchor class]]) {
-      // We only want anchors of type plane.
-      continue;
-    }
-    ARPlaneAnchor *plane = (ARPlaneAnchor *)anchors[i];
-    NSString *identifierStr = [NSString stringWithFormat:
-                           @"%i",
-                           (int)plane.identifier];
-    if (i < anchors.count - 1) {
-      identifierStr = [identifierStr stringByAppendingString:@","];
-    }
-    removedStr = [removedStr stringByAppendingString:identifierStr];
-  }
-  removedStr = [removedStr stringByAppendingString:@"]"];
-
-  [self dispatchVRDisplayPlaneEvent:@"planesremoved" planes:removedStr];
+  [self dispatchVRDisplayPlaneEvent:@"planesremoved" planes:[self getPlanesString:anchors]];
```

Comment:
What is the purpose of this change? To reuse as much code as possible? Think that the structure that needs to be created is more complex with unnecessary data (matrices). In ARCore, the same approach (ids only) is being used right?

Reply:
It means I don't have to intercept the event in javascript and then rebuild a new event, which seems very bizarre to me. I think this is the cleanest solution.